target/snitch_cluster/sw/run.yaml

```diff
@@ -11,6 +11,8 @@ runs:
   - elf: tests/build/data_mover.elf
   - elf: tests/build/dma_empty_transfer.elf
   - elf: tests/build/dma_simple.elf
+  - elf: tests/build/dma_mchan.elf
+    simulators: []
```

Comment:
I removed it

Reply:
I remember again why I put it there. If it is not in the list, it does not compile the test binary. I now added a commit https://github.com/pulp-platform/snitch_cluster/pull/144/commits/5e29bbb60b7cb94004dc8893860437004c349a58 to compile specific tests. Let me know if you don't like it.

Reply:
Compiling all tests makes sense to me as well. But we still need a way to run specific tests for specific HW configurations. For instance the `dma_mchan` test only works if with a multi-channel DMA configuration.

Reply:
Yes, that's provided by the `run.py` script, with the input YAML files. So that would stay the same. Simply instead of retrieving the list of tests to compile from the list of tests to run, we keep the two things independent.

Reply:
That makes sense 👍

Reply:
This is addressed in https://github.com/pulp-platform/snitch_cluster/pull/144/commits/4f9141b0221d4d4c9167b9e1a0a6672074dbad2d